stories/intro/Overview.stories.tsx

```diff
@@ -30,18 +31,23 @@ export const Overview: Story = () => {
   );
 
   React.useEffect(() => {
-    const hideAddonsButton = parent.document.querySelector(
-      'button[title="Hide addons [A]"]',
+    const shortcutsButton = parent.document.querySelector(
+      'button[title="Shortcuts"]',
     ) as HTMLButtonElement;
+    shortcutsButton?.click();
 
-    const isPanelOpen = !!parent.document.querySelector(
-      '#storybook-preview-wrapper',
-    )?.parentElement?.parentElement?.style.height;
+    setTimeout(() => {
```

Comment:
Sure it always works with timeout? Usually timeout solutions are not stable...

Reply:
I tested in Chrome and Firefox in incognito mode and in both light and dark themes.

Will deploy storybook after merging this so others can test.